package.json

```
@@ -28,6 +28,7 @@
   "devDependencies": {
     "github-changes": "^1.1.2",
     "lerna": "^3.22.1",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
+    "playwright": "^1.49.0"
```

```suggestion
    "playwright": "1.49.0"
```
If you are pinning playwright to `1.49.0` in the scripts, you prob want to pin playwright in the package.
`^1.49.0` will accept anything up to `2.x.x`

---

CI was updated in https://github.com/less/less.js/pull/4319 so I backed out the CI changes in this PR and am resolving the conversation for now.